src/pages/jaewonhwang/Main/Main.scss

```
@@ -0,0 +1,275 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css");
+
+.mainJW {
+  .nav {
+    width: 85%;
+    height: 80px;
+    font-size: 25px;
+    font-family: "Lobster", cursive;
+    margin: 0 auto;
+    align-items: center;
+    @extend %flex_between;
+
+    .main_search {
+      width: 210px;
+      height: 30px;
+      text-align: center;
+      border: 1px solid #ddd;
+      background-color: rgba(0, 0, 0, 0.03);
+      border-radius: 3px;
+      display: flex;
+      align-items: center;
+
+      .fa-search {
+        font-size: 15px;
+        margin-left: 70px;
+      }
+
+      input {
+        width: 100px;
+        height: 28px;
+        margin-left: 10px;
+        border: 0;
+        background-color: transparent !important;
+      }
+    }
+
+    .nav_icons {
+      width: 18%;
+      height: 30px;
+      align-items: center;
+      @extend %flex_between;
+
+      span img {
+        width: 35px;
+        border-radius: 50%;
+      }
+    }
+  }
+
+  .main_all {
+    width: 100%;
+    padding-top: 10px;
+    border-top: 1px solid #ccc;
+    background-color: rgba(0, 0, 0, 0.03);
+
+    .main_feed {
+      width: 85%;
+      margin: 0 auto;
+      @extend %flex_between;
+
+      .feeds {
+        width: 65%;
+
+        .story {
+          background: #fff;
+          border: 1px solid #ccc;
+          padding: 20px 0 10px;
+          border-radius: 3px;
+          ul {
+            overflow: hidden;
+            @extend %flex_between;
+            li {
+              width: 70px;
+              height: 80px;
+              margin: 0 12px;
+              text-align: center;
+              list-style: none;
+              img {
+                width: 50px;
+                border-radius: 50%;
+              }
+              p {
+                overflow: hidden;
+              }
```

```suggestion
        .story {
          background: #fff;
          border: 1px solid #ccc;
          padding: 20px 0 10px;
          border-radius: 3px;
          
          ul {
            overflow: hidden;
            @extend %flex_between;
            
            li {
              width: 70px;
              height: 80px;
              margin: 0 12px;
              text-align: center;
              list-style: none;
              
              img {
                width: 50px;
                border-radius: 50%;
              }
              
              p {
                overflow: hidden;
              }
```
가독성을 위해서 다른 tag, className, id 들의 스타일링 사이사이에 한줄 공백 넣어주시면 좋을 것 같네요

---

다른 부분도 마찬가지 입니다~!